flink-runtime/src/test/java/org/apache/flink/runtime/rest/FileUploadHandlerITCase.java

```diff
@@ -56,26 +56,26 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
 
 /**
  * Tests for the {@link FileUploadHandler}. Ensures that multipart http messages containing files
  * and/or json are properly handled.
  */
-public class FileUploadHandlerITCase extends TestLogger {
+class FileUploadHandlerITCase {
```

Comment:
There are several tests in this class with `public` modifier which could be hardened